test/network_analysis/conservation_laws.jl

```diff
@@ -224,8 +224,8 @@ let
     oprob2 = ODEProblem(osys, u0, 10.0, ps_2)
 
     # Checks that the solutions generates the correct conserved quantities.
-    sol1 = solve(oprob1; saveat = 1.0)
-    sol2 = solve(oprob2; saveat = 1.0)
+    sol1 = solve(oprob1, Tsit5(); saveat = 1.0)
+    sol2 = solve(oprob2, Tsit5(); saveat = 1.0)
```

Comment:
Same -- I'm going to stop commenting on this now, but there shouldn't be any `solve` commands that get modified by this PR. Just the dependencies and the `using`s.